styles/widgets/material/textEditor.material.less

```diff
@@ -316,12 +316,7 @@
     .dx-clear-button-area {
         width: @MATERIAL_TEXTEDITOR_ICON_CONTAINER_SIZE;
         min-width: @MATERIAL_TEXTEDITOR_ICON_CONTAINER_SIZE;
-        left: 4px;
-
-        .dx-rtl& {
-            right: 4px;
-            left: 0;
-        }
+        right: 0;
```

Comment:
Is it OK after `DevExpress.config({ rtlEnabled: true })`?

Reply:
Yes, it's ok